islandora_marcxml.module

```diff
@@ -97,3 +105,17 @@ function islandora_marcxml_islandora_ingest_steps_alter(array &$steps, array &$f
     }
   }
 }
+
+/**
+ * Implements hook_islandora_edit_datastream_registry().
+ */
+function islandora_marcxml_islandora_edit_datastream_registry(AbstractObject $object, AbstractDatastream $datastream) {
+  if ($datastream->id == 'MODS') {
+    return array(
+      'islandora_marcxml_update_mods_from_marcxml' => array(
+        'name' => t('Update MODS from MARCXML'),
```

Comment:
@adam-vessey for sure. Nor edit/not regenerate, match fully what is happening here. I agree on that passing this to the rendered output would make more sense, I just wanted to avoid a bigger reformulation of the whole "edit" clockwork because it would imply not only multiple pulls but also a lot of work for @jordandukart , but still, somehow, not sure if my suggestion is the best, keep, at least, in code somewhere (hopefully also in doc) a clear notion that this thing runs on something extra you provide and gives as result not what you provide but MODS: which is different to any other behaviour we have tied to the rendered "edit" or "regenerate" right now.